src/core/lib/security/security_connector/alts/alts_security_connector.cc

```diff
@@ -215,6 +224,8 @@ grpc_alts_auth_context_from_tsi_peer(const tsi_peer* peer) {
   grpc_auth_context_add_cstring_property(
       ctx.get(), GRPC_TRANSPORT_SECURITY_TYPE_PROPERTY_NAME,
       GRPC_ALTS_TRANSPORT_SECURITY_TYPE);
+  grpc_auth_context_add_cstring_property(
+      ctx.get(), GRPC_TRANSPORT_SECURITY_LEVEL, TSI_ALTS_SECURITY_LEVEL);
```

Comment:
you should use the value from security_level_prop, which came from TSI, rather than hardcoded the value.